providers/edge3/src/airflow/providers/edge3/worker_api/routes/ui.py

```diff
@@ -208,3 +243,51 @@ def delete_worker(
         remove_worker(worker_name, session=session)
     except Exception as e:
         raise HTTPException(status.HTTP_400_BAD_REQUEST, detail=str(e))
+
+
+@ui_router.post(
+    "/worker/{worker_name}/queues/add",
```

Comment:
Can you leverage the REST call options, I'd propose to PUT
```suggestion
@ui_router.put(
    "/worker/{worker_name}/queues",
```

Reply:
Done!